packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -405,6 +417,10 @@ export class StatsAnalyzer extends EventsScope {
       this.statsResults[type].direction = statsItem.currentDirection;
       this.statsResults[type].trackLabel = statsItem.localTrackLabel;
       this.statsResults[type].csi = statsItem.csi;
+      this.extractAndSetLocalIpAddressInfoForDiagnostics(
+        this.successfulCandidatePair?.localCandidateId,
+        this.statsResults?.candidates || this.statsResults?.internal?.candidates
```

Comment:
do we really need the fallback to `this.statsResults?.internal?.candidates`? I don't see it ever being populated anywhere and it's empty when I tried the sample app

Reply:
I think somebody refactored something here recently because it was internal.candidates when i opened this draft PR, and then changed to the candidates.

I will remove the fallback.